cli/testdata/coder_support_--help.golden

```diff
@@ -0,0 +1,18 @@
+coder v0.0.0-devel
+
+USAGE:
+  coder support [flags] <workspace> [<agent>]
+
+  Generate a support bundle to troubleshoot issues.
```

Comment:
This file doesn't seem up-to-date? (And missing the new subcommand.)

Reply:
Oh interesting. Maybe because the command is hidden and it wasn't before?

Reply:
Deleted and it's gone!